style.css

```diff
@@ -248,7 +248,7 @@ nav {
   height: 100vh;
   width: 100vw;
   overflow: auto;
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: rgba(193, 199, 208);
```

Comment:
- [ ] Kindly make sure the backend is blur just like the previous reviewer said. User this property to help you `backdrop-filter: blur(12px);`